### ISSUE TITLE
Move the pickles to a special collection

### ISSUE THREAD

**Issue by ekaf** [nltk/nltk_data#219]
Now that alternative data packages are available for all the pickles,
the question arises: what to do with the old packages?

Simply removing them seems very unsafe for those users who are stuck with an old NLTK version which they cannot upgrade, because they would be forced to look elsewhere to get those packages from dubious sources.

So, what about moving them to a special collection, named for ex. "Pickles"?

**Comment by hteeyeoh** [comment]
means old nltk data will renamed as something else and latest one we can remain using punkt to download instead using punkt_tab? cause we saw some other modules underlying using old nltk lib. if they didnt move forward, they will still using old nltk version that download the data via punkt instead punkt_tab

**Comment by ekaf** [comment]
@hteeyeoh , the collections are xml files that provide thematic lists of nltk packages. I am proposing to move the pickles to a new list, while keeping their current https address, so that nothing breaks.

**Comment by hteeyeoh** [comment]
i see. So this means that for modules that did not upgrade nltk version they can still use punkt lib without triggering the security scan?

**Comment by ekaf** [comment]
Yes, @hteeyeoh, one purpose of this issue is to discuss how to handle the case when users cannot upgrade to a newer NLTK version.

**Comment by hteeyeoh** [comment]
Thanks. May I know when can we have this ready?

**Comment by stevenbird** [comment]
@hteeyeoh this is not a time-critical issue, so no promises. I suggest you use whichever punkt package you need.

**Comment by hteeyeoh** [comment]
Hi @stevenbird , 
Ya understand that. Thanks
